hack/lib/etcd.sh

```diff
@@ -46,12 +48,12 @@ kube::etcd::validate() {
   fi
 
   # validate installed version is at least equal to minimum
-  version=$(etcd --version | grep Version | tail -n +1 | head -n 1 | cut -d " " -f 3)
```

Comment:
Why are we changing this and what does it have to do with ARM?

Reply:
since the first line of the output (etcd --version) looks like below,
```
/go/src/k8s.io/kubernetes/third_party# etcd --version
running etcd on unsupported architecture "arm64" since ETCD_UNSUPPORTED_ARCH is set
etcd Version: 3.4.3
Git SHA: 3cf2f69b5
Go Version: go1.12.12
Go OS/Arch: linux/arm64
```
and this commit (09d4d65b1bdd8ece7e781816921419d6426e2e11) doesn't address them all.

maybe some inline comments or commit message is better?

Reply:
Does it output this on all platforms?

Reply:
except AMD64, `etcd --version | head -n 1 | cut -d " " -f 3` is enough on AMD64, `etcd --version | grep Version | head -n 1 | cut -d " " -f 3` should work on all platforms.

Reply:
it outputs on unsupported platforms (on linux this means anything other than ppc64le and amd64 currently)